runtime/src/pallets_governance.rs

```diff
@@ -0,0 +1,207 @@
+    type MembershipInitialized = Allocations;
+    type MembershipChanged = Allocations;
+}
+
+impl pallet_mandate::Config for Runtime {
+    type Event = Event;
+    type Call = Call;
+    type ExternalOrigin =
+        pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, RootCollective>;
+}
+
+parameter_types! {
+    pub const AmendmentDelay: BlockNumber = 2 * constants::DAYS;
+}
+
+impl pallet_amendments::Config for Runtime {
+    type Event = Event;
+    type Amendment = Call;
+    type Scheduler = Scheduler;
+    type SubmissionOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, TechnicalCollective>;
+    type VetoOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type Delay = AmendmentDelay;
+    type PalletsOrigin = OriginCaller;
+}
+
+parameter_types! {
+    pub const CompanyReserveModuleId: ModuleId = ModuleId(*b"py/resrv"); // 5EYCAe5ijiYfha9GzQDgPVtUCYDY9B8ZgcyiANL2L34crMoR
+}
+
+impl pallet_reserve::Config<pallet_reserve::Instance1> for Runtime {
+    type Event = Event;
+    type Currency = pallet_balances::Module<Runtime>;
+    type ExternalOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type Call = Call;
+    type ModuleId = CompanyReserveModuleId;
+}
+
+parameter_types! {
+    pub const InternationalReserveModuleId: ModuleId = ModuleId(*b"py/rvint"); // 5EYCAe5ijiYfi6GQAEPSHYDwvw4CkyGtPTS52BjLh42GygSv
+}
+
+impl pallet_reserve::Config<pallet_reserve::Instance2> for Runtime {
+    type Event = Event;
+    type Currency = pallet_balances::Module<Runtime>;
+    type ExternalOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type Call = Call;
+    type ModuleId = InternationalReserveModuleId;
+}
+
+parameter_types! {
+    pub const UsaReserveModuleId: ModuleId = ModuleId(*b"py/rvusa"); // 5EYCAe5ijiYfi6MEfWpZC3nJ38KFZ9EQSFpsj9mgYgTtVNri
+}
+
+impl pallet_reserve::Config<pallet_reserve::Instance3> for Runtime {
+    type Event = Event;
+    type Currency = pallet_balances::Module<Runtime>;
+    type ExternalOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, FinancialCollective>;
+    type Call = Call;
+    type ModuleId = UsaReserveModuleId;
+}
+
+parameter_types! {
+    pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) *
+        constants::RuntimeBlockWeights::get().max_block;
+    pub const MaxScheduledPerBlock: u32 = 50;
+}
+
+impl pallet_scheduler::Config for Runtime {
+    type Event = Event;
+    type Origin = Origin;
+    type PalletsOrigin = OriginCaller;
+    type Call = Call;
+    type MaximumWeight = MaximumSchedulerWeight;
+    type ScheduleOrigin = frame_system::EnsureRoot<AccountId>;
+    type MaxScheduledPerBlock = MaxScheduledPerBlock;
+    type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
+}
```

Comment:
Probably should go to something like `util`

Reply:
Done :+1: